### DISCUSSION TITLE
No Metamask connection request

### DISCUSSION THREAD

**Discussion by ughur-mammadzada** [WalletConnect/walletconnect-monorepo#3090] [Developer Support]
I have configured W3M html and I can open the modal, but when I select Metamask or read the QR, nothing happens. If Metamask is locked, the pop-up metamask window opens to unlock and then nothing.

```
const { publicClient } = configureChains(coreChains, [w3mProvider({ projectId })])
const wagmiConfig = createConfig({
    autoConnect: false,
    connectors: w3mConnectors({ projectId, chains: coreChains }),
    publicClient
})
const ethereumClient = new EthereumClient(wagmiConfig, coreChains)
const web3modal = new Web3Modal({ projectId }, ethereumClient)
await web3modal.openModal();
```

Previously I tried using wagmi with Wallet Connect Connector and I could connect by reading the QR code. However Wallet Connect Connector doesn't have metamask for desktop, and it doesn't look like I can give wagmi more than 1 connector, so I couldn't add Injected connector.

```
const walletConnectConnector = new WalletConnectConnector({
    chains: coreChains,
    options: {
      projectId,
      showQrModal: true,
    }
});
const wagmiConnection = await connect({
    connector: walletConnectConnector,
});
```

**Comment by finessevanes** [comment]
If you could provide the following details, it would be helpful in diagnosing the problem:

- The version of the SDK you are using 
- The version of the MetaMask wallet you're using
- A brief description or an example of what you mean by "selecting and reading". For instance, are you trying to select by clicking and reading by scanning?

**Reply by ughur-mammadzada** [reply]
Packages:
```
"@web3modal/ethereum": "^2.4.7",
"@web3modal/html": "^2.6.2",
```

UPD: `npm list @wagmi/core` gives version 1.3.4 which is the dependency of `@web3modal/ethereum`. 

Metamask extension for Firefox: 10.32.0
Metamask mobile app: 7.1.0

By selecting, I mean, clicking on Metamask icon on Desktop section of the modal. It is written as installed. By reading I mean scanning the QR code with Metamask mobile app;
